tests/test_ophys/test_tools_roiextractors.py

```diff
@@ -2032,5 +2033,15 @@ def test_add_summary_images_from_two_planes(self):
             np.testing.assert_almost_equal(image_data, extracted_images_dict[image_name_from_metadata])
 
 
+class TestAddToNwbfile(TestCase):
+
+    def test_add_to_nwbfile(self):
+
+        nwbfile = mock_NWBFile()
+
+        interface = MockImagingInterface()
+        interface.add_to_nwbfile(nwbfile)
```

Comment:
```suggestion
def test_add_to_nwbfile():

    nwbfile = mock_NWBFile()

    interface = MockImagingInterface()
    interface.add_to_nwbfile(nwbfile)
```

Any need for this to be a `TestCase`?